Created a seed generator class along with a driver class for testing.

k-short opened the pull request.

k-short committed:
Created a seed generator class along with a driver class for testing.

hossamkhader commented:
Please pull from upstream first, and then resubmit the pull request.
This should resolve the current conflict.

hossamkhader commented:
Return a null string instead of INPUT_ERROR

k-short commented:
You'll have to show me in class tomorrow.   Why return a null String?

k-short commented:
I can actually see how returning null will be better once other classes are calling the generateSeed() method, that way they can do a null check for a valid seed.  I will get that changed in the morning.

mabranon commented:
Just a suggestion, but maybe pass h instead of r and calculate r in the constructor. n is the length of the subwords, h is the number of holes, r is just n-h. I figure that someone calling the method knows n and h, and r is just kind of shorthand for n-h. 

For another take on the returning null thing, we could throw an IllegalArgumentException. Same kind of idea, but it's more explicit.

k-short committed:
- seedGenerator() now returns a null String for invalid input.
- Paremters for seedGenerator() are now subword length (n) and number of holes (h).
